src/intl/en/page-developers-learning-tools.json

```
@@ -35,6 +35,8 @@
   "page-learning-tools-remix-logo-alt": "Remix logo",
   "page-learning-tools-sandbox": "Code sandboxes",
   "page-learning-tools-sandbox-desc": "These sandboxes will give you a space to experiment with writing smart contracts and understanding Ethereum.",
+  "page-learning-tools-speed-run-ethereum-description": "Use Scaffold-ETH to expedite your Solidity knowledge. When you are ready to test your knowledge, speed run Ethereum.",
```

```suggestion
  "page-learning-tools-speed-run-ethereum-description": "SpeedRunEthereum is a set of challenges to start developing on Ethereum uring Scaffold-ETH",
```
Maybe something like this?

---

yeah that sounds good, I just ripped some text from their website =)